.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy GitHub Pages
+
+on:
+  push:
+    branches:
+      - trunk
+      - add/* # Temporary addition for testing.
```

Comment:
Thinking more, since it's internal, maybe we could have it for all branches? And maybe rename the workflow name to make sure we'll only add internal things here? WDYT?

Reply:
> Should we remove it now? I'm also fine to keep it if we want since the link is not shared anywhere.

I left it because it's also there in wpjobmanager.com, but I am happy removing it too.

> Thinking more, since it's internal, maybe we could have it for all branches? And maybe rename the workflow name to make sure we'll only add internal things here? WDYT?

That's a good point. I just thought of another alternative: We leave only "trunk" and we add the "workflow_dispatch" event too (so we can trigger it manually from any branch if we want to through the Gihub Actions interface). WDYT?

Reply:
> That's a good point. I just thought of another alternative: We leave only "trunk" and we add the "workflow_dispatch" event too (so we can trigger it manually from any branch if we want to through the Gihub Actions interface). WDYT? (Done in https://github.com/Automattic/WP-Job-Manager/commit/cae029105cbfd04d30cbfd80cb4dc593c9579da2)

Nice! It makes sense to me!

> Not sure what you mean with "rename the workflow name". Do you mean the file? github-pages.yml to something like internal-docs-gh-pages.yml?

Yep! But I'm not remembering now why I thought to have only internal things in that workflow. 🤔 So ignore it, and if I remember something or we realize anything we could iterate later.